universal-login-react/src/config/config.ts

```diff
@@ -2,9 +2,9 @@ require('dotenv').config();
 import {ETHER_NATIVE_TOKEN} from '@universal-login/commons';
 
 const developmentConfig = {
-  domains: ['mylogin.eth'],
-  relayerUrl: 'http://localhost:3311',
-  jsonRpcUrl: 'http://localhost:18545',
+  domains: ['poppularapp.test'],
+  relayerUrl: 'https://relayer-rinkeby.herokuapp.com',
+  jsonRpcUrl: 'https://rinkeby.infura.io/v3/b3026fc5137a4bd18e5d5906ed49f77d',
```

Comment:
This should work by default on local host

Reply:
Yeah, that was not supposed to be commited